<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>573</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>582</width>
    <height>573</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>582</width>
    <height>573</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BoringCovid</string>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>562</width>
      <height>533</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>562</width>
      <height>533</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>562</width>
      <height>533</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>176</width>
       <height>418</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>5</y>
       <width>370</width>
       <height>425</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>server</string>
     </property>
     <widget class="QLabel" name="label_remote_address">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>69</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>address</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_remote_port">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>69</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>port</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_password">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>69</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>password</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_method">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>69</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>method</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_remote_address">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>70</y>
        <width>252</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_remote_port">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>140</y>
        <width>252</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_password">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>210</y>
        <width>252</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_method">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>270</y>
        <width>256</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>none</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>aes-128-gcm</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>aes-256-gcm</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>chacha20-ietf-poly1305</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="lineEdit_remark">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>340</y>
        <width>252</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_remakr">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>340</y>
        <width>69</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>remark</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_add">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>430</y>
       <width>87</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>add</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_delete">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>430</y>
       <width>87</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>delete</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_copy">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>465</y>
       <width>87</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>copy</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_move_up">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>500</y>
       <width>87</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>move up</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_move_bot">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>500</y>
       <width>87</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>move down</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>440</y>
       <width>173</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>440</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>local port</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_start">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>500</y>
       <width>87</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_stop">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>500</y>
       <width>87</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>stop</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_apply">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>500</y>
       <width>87</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>apply</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>582</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
